### DISCUSSION TITLE
Save *Custom* Cup Profile templates?

### DISCUSSION THREAD

**Discussion by peregrineroasters** [artisan-roaster-scope/artisan#799] [Q&A] [answered]
Hi, all.

First post on here and really hoping there is a solution to the issue I'm having.

For years, my shop has used paper copies of SCAA cupping sheets with the following score factors:
- Fragrance/Aroma
- Aftertaste
- Flavor
- Acidity
- Body
- Uniformity
- Balance
- Clean Cup
- Sweetness
- Overall

As you can see, there are a total of 10 metrics. . . noticeably different metrics than all of the default cup profile templates that Artisan stores (ie: Sweet Maria's, Intelligentsia, SCCA, etc.)

I am trying my darndest to avoid filing through countless SCAA cupping sheets to find the detailed scores and notes. I know I could create a spreadsheet, but utilizing Artisan's roast reports and spider graphs would be an AMAZING upgrade on "this was an 86.25 point coffee"

Is there any way possible to STORE the "*CUSTOM*" set of metrics/factors? I have been able to do it manually, but it is extremely tedious to create this template each and every time I switch to a different alog file.

Thanks in advance.

Peregrine

**Comment by MAKOMO** [comment, marked as answer]
A typo prevented the *CUSTOM* setup being persisted as intended. Fixed also the typo (SCCA => SCAA) and added the current SCA setup. A new Continous build is already available with this fix:

https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous

Note that currently the Windows build is broken, which should be fixed by tomorrow.

Thanks for reporting!

**Reply by peregrineroasters** [reply]
Thanks so much. Very appreciative of you.

**Comment by MAKOMO** [comment]
This works for all platforms. The Windows build is fixed by now too. The link points to beta version of the next Artisan version that fixes the issue. You can try those out or wait until the next official release of Artisan (most likely end of next month).

**Reply by bigl00z3** [reply]
Nice update. Thank you.
Just thinking out loud.... Would it be nice to have the cupping form score (if filled) also printed in the roast summary comment box  on the main window?
Nico

**Reply by MAKOMO** [reply]
Could suggestion. Just added this. Will be in one of my next commits.
